services/workflows-service/src/health/consts.ts

```diff
@@ -0,0 +1,6 @@
+export const CONTROLLER_NAME = '_health';
+
+export const ROUTES = {
+  LIVE: 'live',
+  READY: 'ready',
+};
```

Comment:
Note this is the syntax for enums as objects.
```suggestion
} as const;
```